src/main/java/com/elytradev/concrete/inventory/gui/client/ConcreteContainerGui.java

```diff
@@ -97,10 +98,6 @@ public boolean doesGuiPauseGame() {
 	
 	@Override
 	protected void keyTyped(char typedChar, int keyCode) throws IOException {
-		if (keyCode == this.mc.gameSettings.keyBindInventory.getKeyCode()) {
-			this.mc.player.closeScreen();
-			return;
-		}
```

Comment:
Note: `GuiContainer` already handles the inventory key bind, so this is not needed.